docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+
+Guidance rules:
+
+* the less powerful party should be protected, i.e., CoC cannot be invoked by lead community member to silence critique of how funds are spent.
+  But, CoC should protect new contributors or early career members from more powerful community members, e.g., in operational or technical disputes.
+* technical content is not invalidated by tone, but tone must remain civil at all times.
+  I.e., valid arguments are not discarded because of tone, but participants in a technical
+  dispute may be punished for tone that is unwelcoming, e.g., name calling etc.
+* conditions that make compliance with certain aspects of CoC more difficult should be taken into account,
+  especially in parties that are the less powerful in a dynamic. Common examples are certain forms of neuroatypicality.
+  A more illustrative but very rare example would be Tourette's with the rare swearing tic that cannot be "turned off".
+* issues of systemic exclusion or discrimination should be addressed when apparent, even at the cost of some community discomfort.
+  E.g., meetings should be moved to less convenient times, or processes made more "formal",
+  if not doing so would exclude others in a discriminatory fashion.
+
+Behaviour that is clearly racist, sexist, etc, is always a CoC violation, and never part of trade-offs.
+
+
+A.2 Guidance on applying the CoC for resource allocation decisions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section deals with resource allocation decisions by ``sktime`` community members,
+and criteria for decision making on resource allocation violating the code of conduct.
+
+Scope
+^^^^^
+
+In scope for this section are all resource allocation decisions that satisfy at least one of the following criteria:
+
+* any decisions that allocate resources primarily obtained due to the ``sktime`` project and the volunteer contributions of the community.
+  This includes research grants awarded directly for the benefit of the ``sktime`` project,
+  commercial opportunities arising from or directly directed to the ``sktime`` project,
+  and any activities that use the ``sktime`` brand as a primary branding.
+  This condition applies even if relevant decisions are made in personal roles, or roles other than ``sktime`` official roles.
+* any decisions by members of the ``sktime`` developer community, in their execution of an official ``sktime`` role.
+  This especially includes core developer, Community Council, and Code of Conduct Committee roles.
+
+Examples for in-scope:
+
+* an academic allocating grant monies from a grant, or a data science consultant providing consultancy services, with a primary ``sktime`` branding
+* hiring decisions for roles that are advertised with a primary ``sktime`` branding
+* spending decisions from ``sktime`` community administered accounts
+* decisions arising from academic collaboration or commercial requests directed to the ``sktime`` community,
```

Comment:
n 3 - fully agree
n 4 - we can only demand this from community leaders (CC). We can't stop our contributors from agreeing to such collaboration if they saw a post in public channel. If our mentee sees academic opportunity posted like "looking for a researcher who has experience with sktime" they're very free to contact that person. If one of our contributors sees that there's a commercial request - same.

Reply:
> we can only demand this from community leaders

Sure, agreed on the community leader point. I clarified it in the scope statement that this is basically the scope of jurisdiction here, also including core developers and events (e.g., people intentionally destroying job ad posters around a "women in data science" special event could be tried for opportunity discrimination)

Reply:
> If our mentee sees academic opportunity posted like "looking for a researcher who has experience with sktime" they're very free to contact that person

Mentees in this scenario do not make hiring decisions, so they would not be impacted. They *get* hired (or not), so you would police the person who does the hiring, and only if they are part of `sktime` or do it on behalf of `sktime` (e.g., panelists on an `sktime` grant who are not elected `sktime` members but have delegated authority)

Reply:
Re consultancy, a mentee would be impacted if they say "I'm providing consultanty *on behalf of `sktime`*" and create the appearance of officiality.

If they put on their CV that they were an `sktime` mentee and say they have experience, but bill on a personal (not `sktime` branded company or activity), it's completely fine and they are not subject to the jurisdiction.

See the point on `sktime` branding above on the distinction.

Reply:
Ok in my opinion a mentee does make a decision - to take the opportunity, and are free to do so. They can make a decision to advertise this opportunity further, right? If they take it themselves instead I see it as absolutely ok scenario

Reply:
of course, mentees and other community members can take on published job opportunities or requests for consultancy. They can also say they are contributors to `sktime`, on their CV. The line is at claiming they are "official `sktime` consultants" or the like (in a case where they are not). I would say "`sktime` developer" is already on the wrong side on that line, unless they are a core developer.